lib/ansible/plugins/loader.py

```diff
@@ -505,10 +505,6 @@ def find_plugin(self, name, mod_type='', ignore_deprecated=False, check_aliases=
         from ansible.vars.reserved import is_reserved_name
 
         plugin = self._find_plugin(name, mod_type=mod_type, ignore_deprecated=ignore_deprecated, check_aliases=check_aliases, collection_list=collection_list)
-        if plugin and self.package == 'ansible.modules' and name not in ('gather_facts',) and is_reserved_name(name):
-            raise AnsibleError(
-                'Module "%s" shadows the name of a reserved keyword. Please rename or remove this module. Found at %s' % (name, plugin)
-            )
```

Comment:
This `find_plugin` should just be nuked, and `_find_plugin` renamed to `find_plugin`

Reply:
Heh, yeah, good point, it's pretty no-oppy now, isn't it...